tests/ping.go

```diff
@@ -80,6 +80,33 @@ func PingFromVMConsole(vmi *v1.VirtualMachineInstance, ipAddr string, args ...st
 	return nil
 }
 
+func PingHTTPFromConsole(vmi *v1.VirtualMachineInstance, host string, port int) error {
```

Comment:
is the name right ?
it doesnt ping , unless curl is fine to use as "ping" 

do we need ipv6 support for curl ? (not dual stack even, first for kind, but i guess it would fail if it runs it, unless its skipped on kind).

Reply:
It aligns with the TCP ping that we have in another opened PR. I'm not a big fan of the name. It is mostly to keep it consistent. If the TCP ping name changes, I'm down to change this too.

Not sure about IPv6 support. That will be an issue for the follow-up adding dual-stack support.

Reply:
We should eliminate those awkward functions names its confusing and not reflects what the function actualy does.
Could you paste the link to that PR so we can discuss changing those function names there?

Reply:
> Not sure about IPv6 support. That will be an issue for the follow-up adding dual-stack support.

We are already IPv6 enabled in both production and tests (although some PR/s are still pending merge).
I would not expect future tests added without handling dual-stack in mind from day-one.

> We should eliminate those awkward functions names its confusing and not reflects what the function actualy does.

The name is not confusing IMO and the other proposals I saw are much worse with an attempt to write a story in the name. I am fine to change "ping" to something that has an equivalent meaning and is short if someone brings it to the table.
And no, "ping" is not bind to ICMP, it is a general word we use even in chat messaging and other contexts to express "connectivity" checks.

Reply:
thx, wondering, the tests are bridge binding right ?
according Quique we dont need to test ipv6 dual stack on those, is it right ?